core/src/test/scala/integration/kafka/api/MetricsTest.scala

```diff
@@ -54,6 +57,12 @@ class MetricsTest extends IntegrationTestHarness with SaslSetup {
 
   @BeforeEach
   override def setUp(testInfo: TestInfo): Unit = {
+    if (testInfo.getDisplayName.endsWith("true")) {
```

Comment:
Wouldn't this enable RemoteStorage for all tests which use `@Parameterized` (and not just for testMetrics)? An alternative way is to add the full test name here OR check for string "systemRemoteStorageEnabled: true".

Reply:
You're right! Currently there's only 1 test in this test suite, but we should make it much clear. Updating now.